app/marko/package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "@storybook/marko",
+  "version": "4.0.0-alpha.1",
```

Comment:
change the version to the latest

Reply:
Some reason when i do that i get below error when i do a yarn bootstrap

> error An unexpected error occurred: "https://registry.yarnpkg.com/@storybook%2fmarko: Not found".

Reply:
You probably need to change it in example dependency as well